docs/installation.rst

```diff
@@ -560,6 +561,156 @@ As mentioned before, the script engine can be started from within ``$SCRIPT_ENGI
 You have to ensure your Init script starts the Node.js executable ``node`` and uses the JavaScript file ``$SCRIPT_ENGINE_HOME/server.js`` as an argument.
 Note, if you use a restricted user to execute the command, make sure the user has appropriate access to the ``$SCRIPT_ENGINE_HOME`` directory.
 
+.. _install-mail-relay:
+
+Installing the Workflow Accelerator Mail Relay
+----------------------------------------------
+.. important:: You only need to install and configure the Mail Relay if you purchased a version of Workflow Accelerator which allows you to use email triggers.
+
+The mail relay is a SMTP server which receives emails and forwards them via HTTP to the Workflow Accelerator web application.
+It is required in order to start a workflow by email.
+If you don't plan on starting workflows by email, you don't have to setup this component and can skip this section.
+
+The complete setup of the mail relay consists of the following steps:
+
+#. Create an email domain for the mail relay
+#. Install and configure the mail relay
+#. Configure the web application
+
+Create an email domain for the mail relay
+`````````````````````````````````````````
+In order to work properly, the mail relay needs to receive all relevant emails.
+The used email addresses are created dynamically based on the ID of the workflow that should be started.
+The domain used for the email addresses can be chosen by you and has to be configured in the system. 
+We advise that you create a specific subdomain which is different from your normal email domain and redirect any incoming emails to the mail relay.
+
+For example if you use the email domain ``mail.yourcompany.com``, a typical email address for the workflow system would look like this: ::
+
+    process-5702854fd1dfff250dc57994@mail.yourcompany.com
+
+How you route the emails to the mail relay is up to you and depends on your infrastructure.
+If the server which runs the mail relay is publicly available, you can simply setup a respective MX record.
```

Comment:
Link to https://en.wikipedia.org/wiki/MX_record to clarify that it’s not our jargon